### ISSUE TITLE
Upcoming test suite breakage to to changes in file

### ISSUE THREAD

**Issue by cbiedl** [ahupp/python-magic#283]
Heads-up: This commit in file:

```commit ab6efaf559950d70767b46f69ea05f70f99dfa25
Author: Christos Zoulas <christos@zoulas.com>
Date:   Wed Dec 21 15:55:52 2022 +0000

    Add more extensions and remove truncated stuff from size (Joerg Jenderek)
```

introduced more extensions for gz-compressed files, breaking the test suite. Consider relaxing the test as for example:

```
--- a/test/python_magic_test.py
+++ b/test/python_magic_test.py
@@ -134,7 +134,7 @@
             self.assert_values(m, {
                 # some versions return '' for the extensions of a gz file,
                 # including w/ the command line.  Who knows...
-                'test.gz': ('gz/tgz/tpz/zabw/svgz', '', '???'),
+                'test.gz': ('gz/tgz/tpz/zabw/svgz/adz/kmy/xcfgz', 'gz/tgz/tpz/zabw/svgz', '', '???'),
                 'name_use.jpg': 'jpeg/jpg/jpe/jfif',
             })
         except NotImplementedError:
```

Kind regards,

**Comment by david-geiger** [comment]
Hi,

Same build issue here on Mageia Linux:

test/python_magic_test.py:53: in assert_values
    self.assertIn(value, expected_value)
E   AssertionError: 'gz/tgz/tpz/zabw/svgz/adz/kmy/xcfgz' not found in ('gz/tgz/tpz/zabw/svgz', '', '???')

**Comment by david-geiger** [comment]
I confim that the proposed patch work and fixes tests.

**Comment by cbiedl** [comment]
Fixed in commit 4ffcd59
